src/Streamly.hs

```diff
@@ -218,20 +211,32 @@ module Streamly
 where
 
 import Data.Semigroup (Semigroup(..))
-import Streamly.Internal.Data.SVar (MonadAsync, Rate(..))
-import Streamly.Internal.Data.Stream.Ahead
-import Streamly.Internal.Data.Stream.Async hiding (mkAsync)
-import Streamly.Internal.Data.Stream.Combinators
-import Streamly.Internal.Data.Stream.Parallel
+import Streamly.Internal.Data.SVar (MonadAsync)
+import Streamly.Internal.Data.Stream.Ahead (Ahead, AheadT)
+import Streamly.Internal.Data.Stream.Async
+       ( Async, AsyncT, WAsync, WAsyncT, (<|))
+-- import Streamly.Internal.Data.Stream.Combinators
+import Streamly.Internal.Data.Stream.Parallel (Parallel, ParallelT)
 import Streamly.Internal.Data.Stream.Serial
-import Streamly.Internal.Data.Stream.StreamK hiding (serial)
+       ( InterleavedT, interleaving, Serial, SerialT, StreamT, WSerial
+       , WSerialT, (<=>))
+import Streamly.Internal.Data.Stream.StreamK (Streaming)
 import Streamly.Internal.Data.Stream.Zip
+       ( ZipAsync, ZipAsyncM, ZipSerial, ZipSerialM, ZipStream
+       , zipping, zippingAsync)
 
 import qualified Streamly.Prelude as P
-import qualified Streamly.Internal.Data.Stream.IsStream as IP
 import qualified Streamly.Internal.Data.Stream.StreamK as K
 import qualified Streamly.Internal.Data.Stream.Async as Async
 
+#define GG x=3
+
+#define DEPRECATE_FUNCTION(fn)                                     \
+{-# DEPRECATED fn "Please use 'Streamly.Prelude.fn' instead." #-}; \
```

Comment:
I don't understand what you're saying here, haddock renders them correctly. Do you mean the case where I pass `fn` also in double quotes?

Reply:
This is how haddock shows this:

> Deprecated: Please use fn instead.

Reply:
Must be clang, I'll use a workaround.

Reply:
Its not clang, `Language CPP` is GHC's c pre-processor. You can use external cpp program and use stringizing operator (`#`) but that may also not solve the problem, because after stringizing you will have to concatenate. I do not think it is easy to do, may be impossible.

Reply:
I don't there's any c pre-processor shipped with ghc that it uses. It just picks up the system's cpp AFAICT. There is a package called `cpphs`, and it looks like both cpphs and clang fail in presence of quoting.